.github/workflows/infra_plan.yaml

```diff
@@ -145,13 +145,13 @@ jobs:
         id: plan
         working-directory: ${{ steps.directory.outputs.dir }}
         run: |
-          terraform plan -lock-timeout=3000s -no-color -out=plan.out 2>&1 | grep -v "hidden-link:" | tee tf_plan_stdout.txt
+          terraform plan -lock-timeout=3000s -no-color -out=plan.out 2>&1 | tee tf_plan_stdout.txt
```

Comment:
thoughts: I guess we should maintain this filter here since action logs are public as well